opal/mca/accelerator/accelerator.h

```diff
@@ -595,6 +695,10 @@ typedef struct {
     opal_accelerator_base_module_device_can_access_peer_fn_t device_can_access_peer;
 
     opal_accelerator_base_module_get_buffer_id_fn_t get_buffer_id;
+
+    opal_accelerator_base_module_wait_stream_fn_t wait_stream;
```

Comment:
nit I think `sync_stream` is a better name and reflects what the operation actually does.

also suggest moving this line up to L665 after `create_stream`

Reply:
I made that change :+1: